korge/src/commonMain/kotlin/com/soywiz/korge/render/RenderContext.kt

```diff
@@ -46,6 +46,8 @@ class RenderContext constructor(
     val views: Views? = bp as? Views?
 
     var debugAnnotateView: View? = null
+    var debugExtraFontScale : Double = 1.0
+    var debugExtraFontColor : RGBA = Colors.WHITE
```

Comment:
Downloaded the PR, and this line seems to fail because RGBA is not found.
You should add a `import com.soywiz.korim.color.*`